ptr.py

```diff
@@ -979,6 +992,9 @@ def main() -> None:
     parser.add_argument(
         "-d", "--debug", action="store_true", help="Verbose debug output"
     )
+    parser.add_argument(
+        "--force", action="store_true", help="Force any disabled tests suites to run"
```

Comment:
Something like `--run-disabled` feels like it would be more explicit and self-explanatory, and also match internal usage.